test/e2e/cli_test.go

```diff
@@ -470,7 +470,12 @@ func (s *CLISuite) TestWorkflowLint() {
 	})
 }
 
-func (s *CLISuite) TestWorkflowRetry() {
+func (s *CLISuite) TestWorkflowRetryNoPersistence() {
```

Comment:
rather than rename, move this test to the new file

Reply:
There are two tests now, one with persistence and one without. This is without and still has behavior to be tested here. It shouldn't be moved out.